backend/src/documentation/endpoints/users/users-delete.yaml

```
@@ -0,0 +1,15 @@
+post:
+  summary: User Delete
+  tags:
+    - users
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: '../../schemas/users/user-delete-request-dto.yaml'
```

Check that all paths are correct
```suggestion
          $ref: '../../schemas/users/users-delete-request-dto.yaml'
```